### DISCUSSION TITLE
Name Change to Lexica Ultralegible

### DISCUSSION THREAD

**Discussion by jacobxperez** [jacobxperez/lexica-ultralegible#116] [Announcements]
**Notice**: Name change only applies to the “**Pro**” version and not the original typeface.

We are excited to announce a new direction for our font project, formerly "_Atkinson Hyperlegible_ **Pro**". To comply with the Open Font License (OFL) and respect the original creator’s rights, we have rebranded our typeface as **Lexica Ultralegible**.

This name change reflects our commitment to enhancing readability and accessibility for all users, particularly low-vision users. "**Lexica**" emphasizes the font's linguistic aspect, reinforcing its purpose as a tool for clear communication. "**Ultralegible**" highlights our dedication to superior legibility, ensuring it is easily read in various contexts.

By adopting Lexica Ultralegible, we aim to maintain the spirit of the original design while establishing a distinct identity. This rebranding allows us to move forward creatively and responsibly. Thank you for your understanding and continued support as we embark on this exciting new chapter!

**Comment by RandomMaerks** [comment]
this is such a massive comeback after weeks of inactivity 👀👀

**Comment by jacobxperez** [comment]
@RandomMaerks 😂😂 sorry I needed to let the other project die for a little bit before I released it under its new name.
